docs/rendering/modelloaders/transform.md

```diff
@@ -1,12 +1,12 @@
 Transform
 ==========
 
-When an [`BakedModel`][bakedmodel] is being rendered as an item, it can apply special handling depending on which transform it is being rendered in. "Transform" means in what context the model is being rendered. The possible transforms are represented in code by the `ItemTransforms$TransformType` enum. There are two systems for handling transform: the deprecated vanilla system, constituted by `BakedModel#getTransforms`, `ItemTransforms`, and `ItemTransform`, and the Forge system, embodied by the method `IForgeBakedModel#applyTransform`. The vanilla code is patched to favor using `applyTransform` over the vanilla system whenever possible.
+When an [`BakedModel`][bakedmodel] is being rendered as an item, it can apply special handling depending on which transform it is being rendered in. "Transform" means in what context the model is being rendered. The possible transforms are represented in code by the `ItemDisplayContext` enum. There are two systems for handling transform: the deprecated vanilla system, constituted by `BakedModel#getTransforms`, `ItemTransforms`, and `ItemTransform`, and the Forge system, embodied by the method `IForgeBakedModel#applyTransform`. The vanilla code is patched to favor using `applyTransform` over the vanilla system whenever possible.
 
-`TransformType`
+`ItemDisplayContext`
 ---------------
 
-`NONE` - Unused.
+`NONE` - Used for the display entity and by Forge when a `Block`'s `RenderShape` is set to `#ENTITYBLOCK_ANIMATED`.
```

Comment:
Should this note that it is used by the display entity by default when no other item display context is set?

Reply:
Sure, the display entity is for debugging purposes.